### DISCUSSION TITLE
Load A Folder For Batch Encoding

### DISCUSSION THREAD

**Discussion by 54x1** [cdgriffith/FastFlix#552] [Ideas]
Is there any way to add a folder with everything inside encoded with the same settings??

**Comment by cdgriffith** [comment]
Hi @54x1, yes that is possible! 

Select one video from that folder, and modify the settings as you want for the encoding of all of them, then save that as a profile. After that profile is created and selected, go to File > Load Directory.  There will be a new pop-up that allows you to select a folder, and see all the videos that will be encoded!

**Comment by 54x1** [comment]
i think what i ment was the problem lies in the audio set up i cant select/deselect the audio tracks across them or have the same audio conversion settings id have to set them manually

**Reply by 54x1** [reply]
@cdgriffith for example if i wanted to convert the audio to "aac", "stereo downmix"  to bitrate "192khz" for all the audios of the folder it doesn't work ?

**Reply by cdgriffith** [reply]
You have to set that up when creating the profile itself: 
![Screenshot 2024-04-27 113431](https://github.com/cdgriffith/FastFlix/assets/3275435/a1be87dc-a1d7-4cc7-90d9-83ae4091cad3)
